distributed/config.py

```diff
@@ -43,16 +42,21 @@
     "tcp-timeout": "distributed.comm.timeouts.tcp",
     "default-scheme": "distributed.comm.default-scheme",
     "socket-backlog": "distributed.comm.socket-backlog",
-    "recent-messages-log-length": "distributed.comm.recent-messages-log-length",
     "diagnostics-link": "distributed.dashboard.link",
     "bokeh-export-tool": "distributed.dashboard.export-tool",
     "tick-time": "distributed.admin.tick.interval",
     "tick-maximum-delay": "distributed.admin.tick.limit",
-    "log-length": "distributed.admin.log-length",
     "log-format": "distributed.admin.log-format",
     "pdb-on-err": "distributed.admin.pdb-on-err",
     "ucx": "distributed.comm.ucx",
     "rmm": "distributed.rmm",
+    # log-length aliases
+    "transition-log-length": "distributed.admin.log-length",
```

Comment:
`distributed.admin.log-length` is the length of the _actual logs_ we keep. There is even an option `admin.log-format`.
Some users care about this (it enables client.get_logs() even if it is just partial logging).

The logs we intend to cut are low level debug logs and I think we should distinguish this

Reply:
Moved everything else to `distributed.admin.low-level-log-length`